setup.py

```diff
@@ -42,7 +42,8 @@
         'django_readonly_field',
     ],
     include_package_data=True,
-    install_requires=[],
+    install_requires=["Django>=1.8,<1.11"],
```

Comment:
Compatibility is not yet assured for django 1.11 and/or master. Could add this, though, but in a separate PR.